services/brig/src/Brig/TURN/API.hs

```diff
@@ -61,17 +60,17 @@ getCallsConfig (_ ::: _ ::: _) = json <$> lift newConfig
     newConfig = do
         env  <- liftIO =<< readIORef <$> view turnEnv
         let (sha, secret, tTTL, cTTL, prng) = (env^.turnSHA512, env^.turnSecret, env^.turnTokenTTL, env^.turnConfigTTL, env^.turnPrng)
-        uris <- liftIO $ randomize (unsafeRange 2) (env^.turnServers)
+        uris <- List1.sort <$> liftIO (randomize (env^.turnServers))
```

Comment:
The purpose of `List1.sort <$> liftIO (randomize` isn't very clear for people not knowing that we wish to sort by scheme and transport, but randomize by host and port - perhaps add a comment here?

Reply:
👍